File: fruitRev1.js
const fruits = require('./fruits.json');

const color_lists = (arr) => {
	let colors = {}
	for (fruit in arr) {
		let color = arr[fruit]
		if (fruit in colors == false) {
			colors[fruit] = [color]
		} else {
			colors[fruit].push(color)
		}

	}
	return colors
}

console.log(color_lists(fruits))

File: fruit_rev1.py
import json

def rev_maker():

    with open("fruits.json") as f:
        fruits = json.load(f)
        color_lists = {}
        for fruit in fruits:
            color = fruits[fruit]
            if color not in color_lists:
                color_lists[color] = [fruit]
            else:
                color_lists[color].append(fruit)
        return print(color_lists)

rev_maker()
